source/configuration/program-detail-page.md

```diff
@@ -268,15 +268,15 @@ const table = {
 const GET_PROGRAM_DETAIL_DATA_QUERY = gql`{
   '<Your GraphQL query>'' {
     '<Data fields returned by your GraphQL API query>'
-  ... 
+  ...
  }
 }
 ```
 
 ### Adding columns to the Program Detail Page Table.
 You can add up to 10 columns in the Program Detail Page Table. If you add more than 10 columns, **Bento will display the first 10 columns without an error or warning message**. The top-down order of columns will be displayed left to right on the UI.
```

Comment:
Remove "You", replace with "Up to 10 columns can be added to the Program Details page table. If more than 10 columns are added..."